secrets.tf

```diff
@@ -0,0 +1,66 @@
+locals {
+  // Since Beanstalk does not have secret injection, we are going to add a list of env vars mapping the secret ids
+  // e.g. POSTGRES_URL => POSTGRES_URL_SECRET_ID = <secret-id>
+  app_secret_ids  = { for key in local.secret_keys : "${key}_SECRET_ID" => aws_secretsmanager_secret.app_secret[key].id }
+  app_secret_arns = [for key in local.secret_keys : aws_secretsmanager_secret.app_secret[key].arn]

```

Comment:
This looks like it's not used. I believe you can remove it.

Reply:
`app_secret_arns` ends up being used in the policy below to ensure we have access to the created keys.

Reply:
See the comment (https://github.com/nullstone-modules/aws-beanstalk-app/pull/8/files#r1443515000) below. This is a duplicate of `local.secret_arns`.

Reply:
Ahhh, excellent catch!